test/parallel_api/ranges/all_any_of_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,54 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+
+#include "support/pstl_test_config.h"
+
+#if _ENABLE_RANGES_TESTING
+#include <oneapi/dpl/ranges>
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ENABLE_RANGES_TESTING
+    constexpr int max_n = 10;
+    int data1[max_n]     = {-1, 1, -1, 3, 4, 5, 6, -1, 8, 9};
+    int data2[max_n]     = {0, 2, 4, 6, 8, 10, 12, 14, 16, 18};
+
+    auto lambda = [](auto i) { return i%2 == 0; };
+
+    bool res1 = 0, res2 = 0;
+    using namespace oneapi::dpl::experimental::ranges;
+    {
+        sycl::buffer<int> A(data1, sycl::range<1>(max_n));
+        sycl::buffer<int> B(data2, sycl::range<1>(max_n));
+
+                                       
+        res1 = any_of(TestUtils::default_dpcpp_policy, views::all(A), lambda);
+        res2 = all_of(TestUtils::default_dpcpp_policy, B, lambda);
```

Comment:
`none_of` is not covered here

Reply:
This algorithm is not targeted on the closer release. So, there is no a test so far.

Reply:
I believe it requires just to add one line to call algo and one line to check results. I think this is all that we need to support `none_of`.

Reply:
I disagree. Rather, it make sense to remove from the path....  

"call algo and one line to check results" - it is called a "test" :-)  In that case we have to be agreed that we are extending our plans to the closer release and add new ones into the JIRA ticket.

Reply:
ok. I have no objections on removing algos from the patch